src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -136,11 +136,11 @@ public boolean equals(Object other) {
      */
     public static class EditPropertyDescriptor {
         private Name name;
-        private Seller seller;
         private Address address;
-        private Price price;
+        private Name seller;
```

Comment:
Can we prefix all the `Person` attributes with `seller`? So something like 
```
private Name sellerName;
private Phone sellerPhone;
private Email sellerEmail;
```

Reply:
Yea that sounds good. Will do.